Description: Use googletest from Debian archive
Author: Shengqi Chen <harry@debian.org>
Forwarded: not-needed
Last-Update: 2025-01-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/gtests/CMakeLists.txt
+++ b/tests/gtests/CMakeLists.txt
@@ -19,7 +19,8 @@
     add_definitions(-DGTEST_HAS_SEH=0)
 endif()
 
-add_subdirectory(${PROJECT_SOURCE_DIR}/third_party/gtest gtest)
+find_package(GTest REQUIRED)
+add_library(dnnl_gtest ALIAS GTest::gtest)
 
 set(APP_NAME "gtest")
 set(MAIN_SRC_GTEST ${CMAKE_CURRENT_SOURCE_DIR}/main.cpp)
@@ -35,7 +36,6 @@
 endif()
 
 include_directories_with_host_compiler(${CMAKE_CURRENT_SOURCE_DIR}
-                                       ${CMAKE_CURRENT_SOURCE_DIR}/gtest
                                        ${CMAKE_CURRENT_SOURCE_DIR}/in
                                        ${CMAKE_CURRENT_SOURCE_DIR}/../../src
 )
